### DISCUSSION TITLE
CLANGD lsp server is changing neovim themes to un desirable themes

### DISCUSSION THREAD

**Discussion by xander412** [NvChad/NvChad#2946] [Q&A]
I am using clangd as an lsp server, even after disabling all the checks the nvchad theme is getting effected. I turned off all the clangd checks by --clang-tidy-checks=-* .
 require'lspconfig'.clangd.setup {
 cmd = { "clangd", "--background-index=false", "--compile-commands-dir=build", "--clang-tidy-checks=-*" },
 
  21 require'lspconfig'.clangd.setup {
  22   cmd = { "clangd", "--background-index=false", "--compile-commands-dir=build", "--clang-tidy-checks=-*" },                               
  23 
  24   init_options = {
  25     usePlaceholders = false,                               
  26     completeUnimported = false,                               
  27     clangdFileStatus = false,                               
  28     indexing = false  -- This option may not be available in all versions                               
  29   },
  30 
  31   -- Disable diagnostics
  32   handlers = {
  33     ["textDocument/publishDiagnostics"] = function() end                               
  34   }
  35 }
  36 .

Even after this nvchad themes are getting effected. 
I need clangd only for autocompletion. Apart from this I don't need anything else from this lspserver

**Comment by siduck** [comment]
show images
